packages/amplify-console-integration/src/setup-profile.ts

```diff
@@ -0,0 +1,69 @@
+import { getProfileName } from './util';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as ini from 'ini';
+import * as os from 'os';
+import * as dotenv from 'dotenv';
+
+function setupAWSProfile() {
+  dotenv.config();
+
+  const dotAWSDirPath = path.normalize(path.join(os.homedir(), '.aws'));
+  const credentialsFilePath = path.join(dotAWSDirPath, 'credentials');
+  const configFilePath = path.join(dotAWSDirPath, 'config');
+  const profileName = getProfileName();
+
+  fs.ensureDirSync(dotAWSDirPath);
+
+  let credentials = {};
+  let config = {};
+  if (fs.existsSync(credentialsFilePath)) {
+    credentials = ini.parse(fs.readFileSync(credentialsFilePath, 'utf-8'));
+  }
+  if (fs.existsSync(configFilePath)) {
+    config = ini.parse(fs.readFileSync(configFilePath, 'utf-8'));
+  }
+
+  let isCredSet = false;
+  Object.keys(credentials).forEach(key => {
+    const keyName = key.trim();
+    if (profileName === keyName) {
```

Comment:
As far as I can tell, we can skip this extra assignment as it's only used once.
```suggestion
    if (profileName === key.trim()) {
```